### PR TITLE
RD-3175 Added modal close confirmation dialog

### PR DESCRIPTION
**Pull request by kubama**
## Description
Added 'Getting started' modal close confirmation dialog.

## Screenshots / Videos
image

## Checklist

- [X] My code follows the UI Code Style.
- [x] I verified that all tests and checks have passed.
- [X] I verified if that change requires a documentation update.
- [X] I added proper labels to this PR.

## Tests
https://jenkins.cloudify.co/view/UI/job/Stage-UI-System-Test/1282/ - only 'getting started' spec

## Documentation
N/A

### CODE REVIEW

File: app/components/GettingStartedModal/GettingStartedModal.tsx [hunk 1]
```diff
@@ -116,8 +118,14 @@ const GettingStartedModal = () => {
         EventBus.trigger('secrets:refresh');
         setInstallationProcessing(false);
     };
-    const handleModalClose = async () => {
-        await modalOpenState.closeModal(modalDisabledChecked);
+    const handleModalClose = () => {
```
Comment (qooban) [inline app/components/GettingStartedModal/GettingStartedModal.tsx:121]: async/await construct is not necessary?

**Review by qooban — COMMENTED** [review]
Added one comment.

There are linter issues to fix:
```
[2021-09-16T08:08:25.417Z] /home/jenkins/agent/workspace/75-getting-started-close-confirm/test/app/components/GettingStartedModal/GettingStartedModal.tsx
[2021-09-16T08:08:25.417Z]    8:39  error  '/home/jenkins/agent/workspace/75-getting-started-close-confirm/test/app/utils/hooks/index.ts' imported multiple times  import/no-duplicates
[2021-09-16T08:08:25.417Z]   27:28  error  '/home/jenkins/agent/workspace/75-getting-started-close-confirm/test/app/utils/hooks/index.ts' imported multiple times  import/no-duplicates
[2021-09-16T08:08:25.417Z] 
[2021-09-16T08:08:25.417Z] ✖ 2 problems (2 errors, 0 warnings)
```

File: app/components/GettingStartedModal/GettingStartedModal.tsx (hunk 1 above)
Comment (kubama) [reply app/components/GettingStartedModal/GettingStartedModal.tsx:120]: I think it served no purpose

Comment (qooban) [reply app/components/GettingStartedModal/GettingStartedModal.tsx:120]: OK, if you say so and tests are passing I don't have more comments.

**Review by qooban — APPROVED** [review]

**Pull request merged**